modules/mobile/app/controllers/mobile/v0/letters_controller.rb

```diff
@@ -49,17 +71,43 @@ def download
 
                      download_service.download_letter(params[:type], request.body.string)
                    end
-        StatsD.increment('mobile.letters.download.type', tags: ["type:#{params[:type]}"], sample_rate: 1.0)
         send_data response,
                   filename: "#{params[:type]}.pdf",
                   type: 'application/pdf',
                   disposition: 'attachment'
       end
 
+      private
+
       def icn
         @current_user.icn
       end
 
+      def validate_format!
+        if params[:format].present? && !params[:format].in?(DOWNLOAD_FORMATS)
+          raise Common::Exceptions::UnprocessableEntity.new(
+            detail: "Format #{params[:format]} not in #{DOWNLOAD_FORMATS}",
+            source: 'Mobile::V0::LettersController'
+          )
+        end
+      end
+
+      def increment_download_counter
+        file_format = params[:format] || 'pdf'
+        StatsD.increment(
+          'mobile.letters.download.type',
+          tags: ["type:#{params[:type]}", "format:#{file_format}"],
+          sample_rate: 1.0
+        )
+      end
+
```

Comment:
This code was stolen from the letters generator controller. 

Note that for EVSS above we use this for getting the body params: `request.body.string`

I don't know with certainty if this is the same or not. It's hard to tell from our specs because our specs just pass the body params through as "params". But given that the letters generator download endpoint is also a post, I suspect it's also using a post body for the params and that they think this will work for the body.

I will attempt to test this in staging to confirm that options pass through.